inst/@symfun/diff.m

```diff
@@ -78,7 +78,7 @@
 
 function z = diff(f, varargin)
 
-  z = diff(f.sym, varargin{:});
+  z = diff(f.symbol, varargin{:});
```

Comment:
Or maybe in this case you could call `diff@sym(f, varargin{:})`. That is supposed to be the proper OO syntax for dispatching to a base class method.